spec/system/rbeapi/api/interfaces_portchannel_spec.rb

```diff
@@ -82,6 +106,19 @@
     end
   end
 
+=begin THIS TEST BREAKS OTHER TESTS
```

Comment:
Fix the test.
the Problem was, that the return value of the parse_lacp_timeout method for portchannel was not defined, therefore the tests didn't work. 
Fixed that.